src/core/client_channel/subchannel.h

```diff
@@ -215,7 +216,10 @@ class Subchannel final : public DualRefCounted<Subchannel> {
 
   channelz::SubchannelNode* channelz_node();
 
-  const grpc_resolved_address& address() const { return key_.address(); }
+  std::string address() const {
+    return grpc_sockaddr_to_string(&key_.address(), true)
```

Comment:
This should use `grpc_sockaddr_to_uri()` instead of `grpc_sockaddr_to_string()`.